src/services/history.js

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+
+import { historyConstants } from "../context";
+
+// History
+export const loadHistoryVideos = async (dispatch) => {
+  try {
+    dispatch({ type: historyConstants.LOADING });
+    const response = await axios.get(
+      `${process.env.REACT_APP_URL}/user/history`,
```

Comment:
Also, I did get this, what is why .env here?

Reply:
Will later replace it with my backend URL.